### ISSUE TITLE
Theoretical time for execution

### ISSUE THREAD

**Issue by wilson428** [kevinschaul/binify#23]
I'm trying to bin 30,000 lat/lng pairs with an -n of 120, and it looks like it will take about an hour. (2 Ghz Macbook Pro, 8GB memory). Is that to be expected? From what I understand of the theory, it should run pretty quickly. May be doing something wrong.

**Comment by kevinschaul** [comment]
Yeah, the algorithm I'm using is very native. It's on my list of things to fix after I finish this semester. Stay tuned!
